examples/helia-browser-verified-fetch/src/App.tsx

```diff
@@ -1,15 +1,19 @@
-import { verifiedFetch } from '@helia/verified-fetch'
+
 import { fileTypeFromBuffer } from '@sgtpooki/file-type'
 import { useCallback, useState } from 'react'
 import { Output } from './Output'
 import { helpText } from './constants'
+import CustomCheckbox from './CustomCheckbox'
+import { getVerifiedFetch } from './utils'
 
-function App (): JSX.Element {
-  const [path, setPath] = useState<string>('')
+function App(): JSX.Element {
+  const [path, setPath] = useState<string>('ipfs://bafybeicklkqcnlvtiscr2hzkubjwnwjinvskffn4xorqeduft3wq7vm5u4')
   const [output, setOutput] = useState<string | JSX.Element>('')
   const [err, setErr] = useState<string>('')
   const [loading, setLoadingTo] = useState<string>('')
   const [controller, setController] = useState<AbortController | null>(null)
+  const [useRecursiveGateways, setUseRecursiveGateways] = useState(true)
+  const [useLibp2p, setUseLibp2p] = useState(false)
```

Comment:
```suggestion
  const [libp2p, setLibp2p] = useState(false)
```

Reply:
I want to make sure this isn't confused for the libp2p instance.

Reply:
makes sense. `use*` is reserved for hooks in a react codebase though. maybe you can use `libp2pInstance` for one or the other?